README.md

```diff
@@ -1,4 +1,4 @@
-# `lexmachine` - Lexical Analysis Framework for Golang
+# `lexmachine` - Lexical Analysis Framewerk for Golang
```

Comment:
? https://www.merriam-webster.com/dictionary/framework

Reply:
ugh, weird. My editor must have added that for some reason. I'll give you a fresh PR without that.